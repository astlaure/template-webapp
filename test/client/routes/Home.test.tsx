import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Home from '../../../src/client/routes/Home';

describe('Home test suite', () => {
  test('Snapshot test matching', () => {
    const component = renderer.create(
      <Home />,
    );
    const tree = component?.toJSON();

    expect(tree).toMatchSnapshot();
  });

  test('DOM rendering', () => {
    const { queryByText } = render(
      <Home />,
    );

    expect(queryByText(/Home Components/i)).toBeTruthy();
  });
});
