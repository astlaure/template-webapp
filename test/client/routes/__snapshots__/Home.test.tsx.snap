// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home test suite Snapshot test matching 1`] = `
<div>
  <h1>
    Home Components
  </h1>
</div>
`;
